minerl/herobraine/env_spec.py

```diff
@@ -27,11 +27,12 @@ class EnvSpec(abc.ABC):
     U_SINGLE_AGENT_ENTRYPOINT = 'minerl.env._singleagent:_SingleAgentEnv'
     U_FAKE_SINGLE_AGENT_ENTRYPOINT = 'minerl.env._fake:_FakeSingleAgentEnv'
 
-    def __init__(self, name, max_episode_steps=None, reward_threshold=None, agent_count=1):
+    def __init__(self, name, max_episode_steps=None, reward_threshold=None, agent_count=None):
         self.name = name
         self.max_episode_steps = max_episode_steps
         self.reward_threshold = reward_threshold
-        self.agent_count = agent_count
+        self.agent_count = 1 if agent_count is None else agent_count
+        self.is_single_agent = agent_count is None

```

Comment:
I think this would make more sense as a method:
```
def is_single_agent(self):
    return self.agent_count is None
```

That would make sure that the single-agent/multi-agent nature of the env_spec properly changes when the agent count is modified after construction, and it makes it much easier in the future if we want to ever move away from the `agent_count is None` structure.